setup.cfg

```
@@ -36,7 +36,7 @@ install_requires =
     requests
     sdcflows ~= 1.3.2
     smriprep ~= 0.7.0
-    tedana >= 0.0.9a1, < 0.0.10
+    tedana >= 0.0.9
```

The following will allow tedana to break API at 0.1 without breaking fmriprep downstream.

```suggestion
    tedana ~= 0.0.9
```

---

Thanks!